From 8bc9d69bd9d2d66d890446af774a60bdca8179b1 Mon Sep 17 00:00:00 2001
From: Mariusz Libera <mariusz.libera@gmail.com>
Date: Wed, 7 Nov 2018 16:28:33 +0100
Subject: [PATCH] fix wxgtk2 and wxgtk3 (#1049)

based on changes suggested by @kosilin
however I'm not sure if antialiassing works properly in wxgtk3 now
---
 src/MathCtrl.cpp | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/MathCtrl.cpp b/src/MathCtrl.cpp
index 6e14afcd..d1e28cff 100644
--- a/src/MathCtrl.cpp
+++ b/src/MathCtrl.cpp
@@ -386,7 +386,7 @@ void MathCtrl::OnPaint(wxPaintEvent &WXUNUSED(event))
 
   wxGCDC antiAliassingDC(dcm);
 
-  #ifdef __WXGTK__
+  #if defined(__WXGTK__) && !defined(__WXGTK3__)
   // Seems like depending on the wxGTK version the antialiassing DC doesn't inherit the
   // scrolling info from the normal DC.
   //
 
